website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -111,9 +111,10 @@ The following changes should be present:
 Commit these changes and push to your remote (assuming your remote is named `origin`):
 
 ```sh
+export RELEASE_VERSION=1.3.0
 git add Makefile changelog website testdata
 git commit -m "Release $RELEASE_VERSION"
-git push -u origin release-$RELEASE_VERSION
+git push -u origin v1.3.x
```

Comment:
This doesn't seem correct. `origin` will be pointing to MY fork. And the branch I create the PR in seems better to be in `release-$RELEASE_VERSION`.

Reply:
yeah, this should REMAIN `release-$RELEASE_VERSION`.